tests/phpunit/bootstrap.php

```diff
@@ -5,6 +5,8 @@
  * @package WooCommerce\Stripe
  */
 
+require_once __DIR__ . '/../../vendor/yoast/phpunit-polyfills/phpunitpolyfills-autoload.php';
```

Comment:
This is needed to run test in previous version of WP (https://make.wordpress.org/core/2021/09/27/changes-to-the-wordpress-core-php-test-suite/)